src/H5Ppublic.h

```diff
@@ -116,21 +116,70 @@ extern "C" {
 /* Define property list class callback function pointer types */
 //! <!-- [H5P_cls_create_func_t_snip] -->
 /**
- * \todo Document me!
+ * \brief Callback function for H5Pcreate_class()
+ *
+ * \param[in] prop_id     The identifier of the property list class being created
+ * \param[in] create_data User pointer to any class creation data required
+ * \return \herr_t
+ *
+ * \details This is function is called when a new property list of the class
```

Comment:
These three entries (lines 125, 149, 172) have "is" both before and after "function".  Probably the first should be removed.